spring-cloud-dataflow-docs/src/main/asciidoc/getting-started-kubernetes.adoc

```diff
@@ -940,29 +1033,42 @@ spec:
   deploymentProperties:
     "spring.cloud.deployer.group": "mystream"
 ----
+====
 
+[[getting-started-kubernetes-create-stream-register-logsink-app]]
+===== Register the `log-sink` Application
 
-10) Register the `log-sink` application version 2.1.0.RELEASE and update your stream to use it
-
-The following example (with output after each command) shows how to register the `log-sink` application and update its version:
-
+Now you need to register the log-sink application.
```

Comment:
So, we need to include the version in the description to highlight it explicitly.

Reply:
Got it.